app/views/publishers/home.html.slim

```diff
@@ -104,9 +51,9 @@ script type="text/html" id="confirm_default_currency_modal_wrapper"
     .row
       .col-md.mb-4
         - if !(@publisher.excluded_from_payout? || @publisher.paypal_locale?(params[:locale]) || @publisher.uphold_connection&.japanese_account? || @publisher.wire_only?)
-          = render partial: "uphold_wallet_panel"
+          = render_async uphold_wallet_panel_publishers_path
```

Comment:
Let's be sure that we show the white panel so that the page makes sense while we're loading this and additionally have the same behavior as the balance panel. 

For bonus points we could create a loading partial that we render anywhere that we're using `render_async`.

I would imagine that it should contain 

- the spinner
- internationalized loading text